rts/lra/lra-filters/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -223,7 +223,7 @@ public void filter(ContainerRequestContext containerRequestContext) {
                         // if there is an LRA present nest a new LRA under it
                         suspendedLRA = incommingLRA;
                         lraTrace(containerRequestContext, suspendedLRA, "ServerLRAFilter before: REQUIRED start new LRA");
-                        newLRA = lraId = startLRA(incommingLRA, method, getTimeOut(method));
+                        newLRA = lraId = startLRA(incommingLRA, method, getTimeOut(transactional));
```

Comment:
could happen that the `transactional` is null? asking as the statement above checks `transactional` of not being `null`

Reply:
On line 180 the method has already returned if transactional is null hence it can never be null at this point in the code.

Reply:
ok, I see now (but it's a bit difficult to understand this code)